imap_processing/swe/tests/test_swe_cem_raw.py

```diff
@@ -0,0 +1,44 @@
+import pandas as pd
+import pytest
+
+from imap_processing import imap_module_directory
+from imap_processing.swe.l0 import decom_swe
+
+
+@pytest.fixture(scope="session")
+def decom_test_data():
+    """Read test data from file"""
+    packet_1_file = (
+        f"{imap_module_directory}/swe/tests/20230927100425_SWE_CEM_RAW_packet.bin"
+    )
+    packet_2_file = (
+        f"{imap_module_directory}/swe/tests/20230927100426_SWE_CEM_RAW_packet.bin"
+    )
+    first_data = decom_swe.decom_packets(packet_1_file)
+    second_data = decom_swe.decom_packets(packet_2_file)
+
+    return first_data + second_data
+
+
+def test_swe_raw_cem_data(decom_test_data):
+    """This test and validate raw data of SWE raw CEM data."""
+    # read validation data
+    test_data_path = f"{imap_module_directory}/swe/tests"
+    raw_validation_data = pd.read_csv(
+        f"{test_data_path}/data_raw.SWE_CEM_RAW_20230927_094839.csv",
+        index_col="SHCOARSE",
+    )
+    data = decom_test_data[0]
```

Comment:
```suggestion
    data0 = decom_test_data[0]
```
I find the `data.data` to be confusing below, so maybe adding a 0 qualifier, or calling this `decom_data` instead instead?